erc20-bridge-token/test/BridgeToken.js

```diff
@@ -399,6 +403,109 @@ describe('BridgeToken', () => {
       ).to.be.equal("0");
     });
 
+    it("Test multiple tokens", async function() {
+      const amountToTransfer = amountToLock;
+      const whitelistTokens = [
+        "near-token1.near",
+        "near-token2.near",
+        "near-token3.near",
+        "near-token4.near",
+      ];
+      const blacklistTokens = [
+        "near-token5.near",
+        "near-token6.near",
+        "near-token7.near",
+        "near-token8.near",
+      ];
+
+      const tokensInfo = [];
+      for (token of whitelistTokens) {
+        tokensInfo.push(await createToken(token));
+        await deposit(token, amountToTransfer, user.address);
+        await BridgeTokenFactory.setTokenWhitelistMode(token, 2);
```

Comment:
Please use constants for whitelist modes (everywhere)

Reply:
cf35ca224be4e69c77bbbb4b295a8a00eb9cd9e7